beta-src/src/components/ui/WDUI.tsx

```diff
@@ -108,6 +114,28 @@ const WDUI: React.FC = function (): React.ReactElement {
     </IconButton>
   );
 
+  const messageTime = useAppSelector((state) => state.game.messages.time);
```

Comment:
In general, when you write something to happen on a useInterval, I don't think this is a good pattern. This will get the messageTime *once* as of the rendering of the component, and then this stale value will get reused in the setInterval potentially multiple times, instead of being the most up to date value.

Also, it causes the component to re-render unnecessarily every time state.game.messages.time changes, even though the only thing you need it for is within the setInterval. Let's just get it directly out of the store, *inside* the callback that happens within the useInterval